stdlib/2and3/distutils/command/config.pyi

```diff
@@ -0,0 +1,83 @@
+from distutils import log as log
+from distutils.ccompiler import CCompiler
+from distutils.core import Command as Command
+from distutils.errors import DistutilsExecError as DistutilsExecError
+from distutils.sysconfig import customize_compiler as customize_compiler
+from typing import Any, Dict, List, Match, Optional, Pattern, Tuple, Union, AnyStr
+
+LANG_EXT: Dict[str, str]
+
+class config(Command):
+    description: str = ...
+    # Tuple is full name, short name, description
+    user_options: List[Tuple[str, Optional[str], str]] = ...
+    compiler: Optional[Union[str, CCompiler]] = ...
+    cc: Optional[str] = ...
+    include_dirs: Optional[List[str]] = ...
+    libraries: Optional[List[str]] = ...
+    library_dirs: Optional[List[str]] = ...
+    noisy: int = ...
+    dump_source: int = ...
+    temp_files: List[str] = ...
+    def initialize_options(self) -> None: ...
+    def finalize_options(self) -> None: ...
+    def run(self) -> None: ...
+    def try_cpp(
+        self,
+        body: Optional[str] = ...,
+        headers: Optional[List[str]] = ...,
+        include_dirs: Optional[List[str]] = ...,
+        lang: str = ...,
+    ) -> bool: ...
+    def search_cpp(
+        self,
+        pattern: Union[Pattern[AnyStr], str],
```

Comment:
AnyStr doesn't make sense here because there's nothing to constrain the TypeVar. From the implementation it looks like it is `Pattern[str]` in Python 3.

Reply:
Sounds good, will update